### PR TITLE
SUB-3576 - Ignore rule - the user cannot edit from "dates" to "never"…

### PR DESCRIPTION
**Pull request by RinaO1234**
… status

### CODE REVIEW

File: service_test.go [hunk 1]
```diff
@@ -426,7 +426,7 @@ func (suite *MainTestSuite) TestPostureException() {
 					},
 				},
 			},
-			expectedIndexes: []int{},
+			expectedIndexes: []int{3},
 		},
 	}
```
Comment (avrahams) [inline service_test.go:431]: Let's add a test that post an exception with  date then put it back with null 
here is code example from our tests 
You can reuse one of the exceptions in ```posturePolicies```

```go
	//put doc without alias - expect the alias not to be deleted
	cluster := testPostDoc(suite, consts.ClusterPath, clusters[0], newClusterCompareFilter)
	alias := cluster.Attributes["alias"].(string)
	suite.NotEmpty(alias)
	delete(cluster.Attributes, "alias")
	w := suite.doRequest(http.MethodPut, consts.ClusterPath, cluster)
	suite.Equal(http.StatusOK, w.Code)
	response, err := decodeResponseArray[*types.Cluster](w)
	if err != nil || len(response) != 2 {
		panic(err)
	}
	suite.Equal(alias, response[1].Attributes["alias"].(string))
	```

Comment (avrahams) [reply service_test.go:431]: Another missing test case is existing exception that where saved with omitempty and does not have the field 
to put such a document in mongo we need to use mongo directly (because the service type has omitempty)

here is a code example how to di that from the tests 
```go
//addGlobal documents to mong db
	defaultCustomerConfig := decode[interface{}](suite, defaultCustomerConfigJson)
	if _, err := mongo.GetWriteCollection(consts.CustomerConfigCollection).InsertOne(context.Background(), defaultCustomerConfig); err != nil {
		suite.FailNow("failed to insert defaultCustomerConfigJson", err.Error())
	}
	```

**Review by avrahams — CHANGES_REQUESTED** [review]

**Review by avrahams — APPROVED** [review]
